PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# Maintainer: Jasur Sadikov <jasur@sadikoff.com>
+pkgname=mud
+pkgver=v1.0.2
```

Comment:
most packages I see in repos are not using v in front

Reply:
We already have tags with `v` prefix :(